### PR TITLE
Fix: p2p suggestions

### PR DESCRIPTION
**Pull request by TheQuantumPhysicist**
Some suggested changes to P2P after a discussion

### CODE REVIEW

**Review by altonen — APPROVED** [review]
LGTM but someone else should probably take a look too

File: p2p/src/message.rs [hunk 1]
```diff
@@ -86,11 +86,11 @@ impl HeaderResponse {
 }
 
 #[derive(Debug, Encode, Decode, Clone, PartialEq, Eq)]
-pub struct BlockResponse {
+pub struct BlocksResponse {
```
Comment (muursh) [inline p2p/src/message.rs:89]: Am I the only one who thinks the pluralisation of this sounds strange? `BlocksResponse` doesn't really make sense

Comment (altonen) [reply p2p/src/message.rs:89]: No, I don't like it either but it was decided that we go with this

Comment (TheQuantumPhysicist) [reply p2p/src/message.rs:89]: It's plural because it's about multiple blocks.

Comment (iljakuklic) [reply p2p/src/message.rs:89]: Could it be something like `BlockListResponse` or `BlockSetResponse`? I find `BlocksResponse` a bit weird too.

Comment (TheQuantumPhysicist) [reply p2p/src/message.rs:89]: I don't mind. Something like BlockListResponse is equally OK for me.

Comment (muursh) [reply p2p/src/message.rs:89]: Yeah fine by me

Comment (muursh) [reply p2p/src/message.rs:89]: I'd suggest the same for `HeadersResponse`

Comment (stanislav-tkach) [reply p2p/src/message.rs:89]: I suppose we should also change the names of the corresponding requests (`BlockRequest` and `HeaderRequest`).

File: p2p/src/net/libp2p/service/pubsub.rs [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn it_works() {
+        assert_eq!(1 + 1, 2);
+    }
+}
```
Comment (stanislav-tkach) [inline p2p/src/net/libp2p/service/pubsub.rs:141]: I'm going to remove that. 🙃

**Review by stanislav-tkach — APPROVED** [review]

**Comment by TheQuantumPhysicist** [comment]
Let's get this merged, as soon as possible, please.

**Comment by altonen** [comment]
I think this is ready, are you guys OK with squash-merging this into master?

**Comment by TheQuantumPhysicist** [comment]
> I think this is ready, are you guys OK with squash-merging this into master?

Quite frankly, I don't care, squash or not. Let's see if someone else has a different opinion. We talk latest about this during the daily.

**Review by muursh — APPROVED** [review]
I'm happy to merge this.

**Pull request merged**